codalab/worker/docker_image_manager.py

```diff
@@ -243,7 +245,11 @@ def image_availability_state(image_spec, success_message, failure_message):
                 def download():
                     logger.debug('Downloading Docker image %s', image_spec)
                     try:
-                        self._docker.images.pull(image_spec)
+                        client = docker.APIClient(base_url='unix://var/run/docker.sock')
```

Comment:
I don't think we need to create a new Docker client here -- can't we just continue to use `self._docker.images`?